apps/api/views/profiles.py

```diff
@@ -72,15 +89,19 @@ def create_students_from_csv(request, version):
                         'klass': request.data.get('klass')
                     }
                 new_student_serializer = TestStudentSerializer(data=data)
-                new_student_serializer.is_valid(raise_exception=True)
-                if new_student_serializer.errors:
-                    return Response({'errors': new_student_serializer.errors}, status=status.HTTP_400_BAD_REQUEST)
-                else:
-                    new_student = new_student_serializer.save()
-                    # If there's something in the student leader column
-                    if len(row) == 7 and contains_valid_team_name and row[6] in ['t', 'T', 'true', 'True', 'TRUE']:
-                        new_student.team.leader = new_student
-                        new_student.team.save()
+                try:
+                    new_student_serializer.is_valid(raise_exception=True)
+                except ValidationError:
+                    if new_student_serializer.errors:
+                        error_dict = dict(new_student_serializer.errors)
+                        error_dict['student'] = {'identifier': [ErrorDetail(f'From top of CSV, {make_ordinal(line_count + 1)} student with email: {str(row[4])}', code='invalid')]}
```

Comment:
Create descriptive error for the identification of the problem row in the CSV.

Reply:
why not just 
```
raise ValidationError(f'From top of CSV, {make_ordinal(line_count + 1)} student with email: {str(row[4])}')
```

Reply:
✅